README.md

```diff
@@ -1,9 +1,47 @@
 # ansible-documentation
-Clone of ansible/ansible with non-docs content removed
 
-Before you build the docs locally, you need to:
+This repository holds the ReStructuredText (RST) source, and other files, for user documentation related to the Ansible package and Ansible core.
 
-```text
-python3 docs/bin/clone-core.py
-```
+> Documentation for modules and plugins that are officially supported by the Ansible core engineering team is available in the [`ansible/ansible`](https://github.com/ansible/ansible) repository.
 
+## Building Ansible community documentation
+
+Complete the following steps to build Ansible community documentation from this repository:
```

Comment:
Are users expected to go through all of these steps each time they want to build documentation?

Reply:
No, I'll add a statement to the end of the readme to point that out.

Reply:
We should remove all of  this. We have a docs section that covers how to build the docs locally. We need to update that section and point to it from this readme. Otherwise we'll be fighting to keep both in sync.  See https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally